checkstyle/suppressions.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<!DOCTYPE suppressions PUBLIC
+        "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+        "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+
+    <!-- TODO -->
```

Comment:
I just accepted what is already in 5.0.x when I rebased -- I think this will make the pint-merge easier. The bit I changed here is the methodLenght.  If there is a better way to do this, let me know!